src/Logger.ts

```diff
@@ -0,0 +1,44 @@
+import { createLogger as createWinstonLogger, format, Logger as WinstonLogger, transports } from 'winston';
+const { configs } = require('./Config');
+
+export class Logger {
+    winstonLogger: WinstonLogger;
+
+    constructor() {
+        this.winstonLogger = createWinstonLogger({
+            level: configs.log.level,
+            format: format.combine(
+              format.timestamp(),
+              format.errors({ stack: true }),
+              format[configs.log.format](),
+            ),
+            transports: [
+                new transports.Console({ level: configs.log.level }),
+            ],
+        });
+    }
+
+    silly(message: string, meta: any = {}): void {
+        this.winstonLogger.silly(message, meta);
```

Comment:
Si je me fie aux types de winston c'est plus un vararg (array) de any qu'il faudrait passer:
https://github.com/winstonjs/winston/blob/master/index.d.ts#L72

À noter que ça affecte aussi l'utilisation où il faut le "spreader" pour passer en paramètre.

S'applique aussi aux autres méthodes de log.

```suggestion
    silly(message: string, ...meta: any[]): void {
        this.winstonLogger.silly(message, ...meta);
```